modules/mono/editor/GodotTools/GodotTools.ProjectEditor/default.csproj.template

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Godot.NET.Sdk/%GODOT_NET_SDK%">
+  <PropertyGroup>
+    <RootNamespace>%ROOT_NAMESPACE%</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="!$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net6.0</TargetFramework>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <PublishAot>true</PublishAot>
+    <PublishAotUsingRuntimePack>true</PublishAotUsingRuntimePack>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <ForceFullAOT>true</ForceFullAOT>
+    <SelfContained>true</SelfContained>
+    <GenerateXcodeProject>true</GenerateXcodeProject>
+    <UseNativeAOTRuntime Condition="'$(UseNativeAOTRuntime)' == ''">true</UseNativeAOTRuntime>
+    <Optimize>false</Optimize>
+    <NativeDebugSymbols>true</NativeDebugSymbols>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(RuntimeIdentifier.Contains('iossimulator'))">
+    <LinkerArg Include="-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk" />
+    <LinkerArg Include="-lc++" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(RuntimeIdentifier.Contains('ios')) and !$(RuntimeIdentifier.Contains('iossimulator'))">
+    <LinkerArg Include="-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk" />
+    <LinkerArg Include="-lc++" />
+    <LinkerArg Include="-licucore" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <RdXmlFile Include="$(IntermediateOutputPath)rd.xml" />
```

Comment:
Instead of this, I'd do:

```suggestion
    <TrimmerRootAssembly Include="GodotSharp" />
    <TrimmerRootAssembly Include="$(TargetName)" />
```

and delete the GenerateRd logic. RD.XML (the obscure dialect of it used in native AOT) is not documented and will be deleted at some point without as much as a breaking change notice. TrimmerRootAssembly is documented, supported, and less ceremony.